src/eventConsumer/collectionEvents/collectionEventConsumer.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  EventTypeString,
+  Collection,
+  CollectionEventPayloadSnowplow,
+} from '../../snowplow/collection/types';
+import { CollectionEventHandler } from '../../snowplow/collection/collectionEventHandler';
+
+// Collection events detail-type in event rule defined in the pocket-event-bridge repo
+// see here .aws/src/event-rules/collection-events/eventConfig.ts
+export const DetailTypeToSnowplowMap: Record<string, EventTypeString> = {
+  'collection-created': 'COLLECTION_CREATED',
+  'collection-updated': 'COLLECTION_UPDATED',
+};
+
+//event bridge payload for Collection
+export type CollectionEventBusPayload = Collection;
+
+export function collectionEventConsumer(requestBody: any) {
+  new CollectionEventHandler().process(getCollectionEventPayload(requestBody));
+}
+
+/**
+ * converts the event-bridge event format to snowplow payload
+ * for a Collection event
+ * @param eventObj event bridge event format
+ */
+export function getCollectionEventPayload(
```

Comment:
calling out for follow-up PR or a different ticket as this PR is in time constraint

the snowplow workflow doesn't check if the mandatory fields are set. if required, we can check for mandatory fields and throw error if they are not present, the upstream function should capture the error and the failing event to the DLQ and we can get alerted